### PR TITLE
Turn remaining top navbar "hover dropdowns" to "click dropdowns"

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
All links in the top navbar are dropdown menus. And you need to click on them to see the dropdown menu... except for these two:

Screenshot from 2022-04-11 15-32-13

For coherency, I turned them from "hover dropdown" into "click dropdown". It'll also allow mobile device users to actually be able to see the dropdown menus for these two items.

### CODE REVIEW

File: templates/header/topbar_begin.html [hunk 1]
```diff
@@ -48,12 +49,17 @@
                     </li>{#
 
                     The Rust dropdown menu
-                    #}<li class="pure-menu-item pure-menu-has-children pure-menu-allow-hover pure-menu-opt">
-                        <a href="https://www.rust-lang.org/" target="_blank" class="pure-menu-link">
-                            Rust
+                    #}<li class="pure-menu-item pure-menu-has-children">
+                        <a href="#" class="pure-menu-link" aria-label="Rust">
+                            <span class="title">Rust</span>
```
Comment (jsha) [inline templates/header/topbar_begin.html:54]: Why not put the title on the `<a>` tag and skip the `<span>`? This title doesn't even show up on hover for me.

**Review by jsha — CHANGES_REQUESTED** [review]
One comment. Otherwise looks good.

File: templates/header/topbar_begin.html (hunk 1 above)
Comment (GuillaumeGomez) [reply templates/header/topbar_begin.html:54]: Just checked and it's indeed unnecessary. Removing it.

**Comment by GuillaumeGomez** [comment]
Updated!

**Review by jsha — APPROVED** [review]
Looks good!

**Comment by GuillaumeGomez** [comment]
I fixed the conflicts.

cc @syphar :)

**Review by syphar — APPROVED** [review]

**Pull request merged**